api/handler/v1beta1/job_spec.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/gofrs/uuid"
```

Comment:
Adding different package for uuid.

Reply:
removed this and use `github.com/google/uuid` instead.